src/api/libcellml/analyserequation.h

```diff
@@ -38,9 +38,10 @@ class LIBCELLML_EXPORT AnalyserEquation
      *  - TRUE_CONSTANT: an equation that computes a true constant, e.g. x = 3;
      *  - VARIABLE_BASED_CONSTANT: an equation that computes a variable-based
      *    constant, e.g. x = y+z where y and z are true constants;
-     *  - RATE: an equation that computes a rate, e.g. d(y)/dt = f(t, x); or
+     *  - RATE: an equation that computes a rate, e.g. d(y)/dt = f(t, x);
      *  - ALGEBRAIC: an equation that computes an algebraic variable, e.g.
-     *    y = f(x).
+     *    y = f(x); or
+     *  - EXTERNAL: an equation that is computed externally.
```

Comment:
The annotations for the enumerations should be done the same way as shown in the Units::StandardUnit enum.
<img width="441" alt="Screen Shot 2022-12-12 at 7 42 44 AM" src="https://user-images.githubusercontent.com/778048/206926319-37015f0d-9e27-4d74-84f5-1f5d37baead9.png">

Reply:
d0d9599e